.circleci/config.yml

```diff
@@ -15,6 +15,15 @@ jobs:
               echo "export DISPLAY=:99" >> $BASH_ENV;
               echo "export OPENBLAS_NUM_THREADS=4" >> $BASH_ENV;
               echo "export PATH=~/.local/bin:$PATH" >> $BASH_ENV;
+              if [ "${DEPS}" == "" ]; then
+                export MNE_ROOT="${PWD}/minimal_cmds";
+                export PATH=${MNE_ROOT}/bin:$PATH;
+                curl https://staff.washington.edu/larsoner/minimal_cmds.tar.gz | tar xz;
+                export LD_LIBRARY_PATH=${MNE_ROOT}/lib:$LD_LIBRARY_PATH;
+                export NEUROMAG2FT_ROOT="${PWD}/minimal_cmds/bin";
+                mne_surf2bem --version;
```

Comment:
why do you need to do this?

Reply:
The `neuromag2ft` binaries live in that `minimal_cmds.tar.gz` file

Reply:
sure but why would you need to check the version of mne_surf2bem? It seems unrelated to PR

Reply:
It's a quick call to ensure that the binaries in that tar file (`mne_surf2bem` is one of these) are properly accessible / on the PATH. It might be possible to do it with `neuromag2ft` directly, which would be potentially less confusing, but also less consistent with the code where this was copy-pasted from, so we have to choose the tradeoff